core/context.py

```diff
@@ -2,10 +2,15 @@
 
 
 def config_context(request):
+    config_identity = request.identity.config_identity if request.identity else None
+
+    reaction_number_css = ""
+    if config_identity and not config_identity.visible_reaction_counts:
+        reaction_number_css = "display: none;"
```

Comment:
This shouldn't be done with a CSS injection like this - instead, there should be a dynamic `<style>` block in the head of base.html that does this instead based on classes.

Reply:
I like that approach a lot better!